articles/tutorials/get-user-information-with-unbounce-landing-pages.md

```diff
@@ -3,64 +3,67 @@
 ### Configuration on Auth0
 
 1. Create an account in [Auth0](https://auth0.com)
```

Comment:
@Annyv2 We don't say that the user should go to the dashboard: `Create an account in [Auth0](https://auth0.com) and navigate to the [dashboard](${manage_url}).`